fronts-client/src/components/inputs/InputImage.tsx

```diff
@@ -257,6 +257,8 @@ interface ComponentState {
 const dragImage = new Image();
 dragImage.src = imageDragIcon;
 
+const regexToCheckGridImage: RegExp = /^https?:\/\/(www\.)?media\.(?:dev-|)guim\.co\.uk\/([0-9a-fA-F]+)\//;
```

Comment:
Thanks to @fredex42 for suggesting regex to check endpoint to distinguish between endpoint poiting to grid or normal s3 uploader (like chef's image)